src/utils/media-utils.js

```diff
@@ -604,10 +604,16 @@ export function parseURL(text) {
 }
 
 export async function resolveMediaInfo(urlString) {
+  //check if url is an anchor hash e.g. #Spawn_Point_1
```

Comment:
Nit: Space after `//` 
```suggestion
  // check if url is an anchor hash e.g. #Spawn_Point_1
```